crowdsourcing/models.py

```diff
@@ -467,4 +468,24 @@ class ConversationRecipient(models.Model):
 class UserMessage(models.Model):
     message = models.ForeignKey(Message)
     user = models.ForeignKey(User)
-    deleted = models.BooleanField(default=False)
\ No newline at end of file
+    deleted = models.BooleanField(default=False)
+
+class File(models.Model):
+    #TODO will need save files on a server rather than in a temporary folder
+    file = models.FileField(upload_to='tmp/')
+    deleted = models.BooleanField(default=False)
+
+    def parse_csv(self):
```

Comment:
spoke to durim about this. consensus was that model methods was the way to go stylistically. because we aren't parsing/deleting the file until we are serializing the module i don't think it really makes sense to have these in the viewset.